### PR TITLE
Led support

### PR DESCRIPTION
**Pull request by Tiliy6**
Added led display to show battery status, indicating a color respectively to its voltage and current. It also blinks to save energy, instead of having it on all the time.

### CODE REVIEW

File: lib/leds/src/leds.h [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+void initLeds();
+void updateLeds();
```
Comment (salmonmolon) [inline lib/leds/src/leds.h:4]: Newline is missing
```suggestion
void updateLeds();

```

File: lib/leds/src/leds.cpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+#include <FastLED.h>
+
+void initLeds(){
+    FastLED.addLeds<WS2812B, DATA_PIN, RGB>(leds, 8);
+}
+
+void updateLeds(){
+
+}
```
Comment (salmonmolon) [inline lib/leds/src/leds.cpp:37]: Newline is missing
```suggestion
}

```

File: lib/leds/src/leds.h (hunk 1 above)
Comment (salmonmolon) [inline lib/leds/src/leds.h:4]: Newline is missing
```suggestion
void updateLeds();

```

File: lib/leds/src/leds.cpp [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    static unsigned long tNextLed = millis();
+    static unsigned long tNextOff;
+
+    if (millis() > tNextLed) {
+        tNextLed = millis() + 2000;
+        if (valor >= 3.8) {
+            leds[0] = CRGB::Green;
+        }
+        else if (valor <= 3.6) {
```
Comment (salmonmolon) [inline lib/leds/src/leds.cpp:23]: Change to english
```suggestion
    float value = analogRead(A0);
    value = value*4.7f/1023;

    static unsigned long tNextLed = millis();
    static unsigned long tNextOff;

    if (millis() > tNextLed) {
        tNextLed = millis() + 2000;
        if (valor >= 3.8) {
            leds[0] = CRGB::Green;
        }
        else if (value <= 3.6) {
```

**Review by salmonmolon — CHANGES_REQUESTED** [review]

**Review by salmonmolon — APPROVED** [review]
Looks ok to me

**Comment by Arnau478** [comment]
I think this was done outside of the PR, on main branch.
Closing!

**Pull request closed without merging**